addons/website_payment/views/website_payment_templates.xml

```diff
@@ -24,59 +24,136 @@
       </xpath>
     </template>
 
-    <template id="pay_methods" name="Payment Methods">
-        <t t-call="website.layout">
-            <t t-set="additional_title">Payment Methods</t>
-            <div class="wrap">
-                <div class="container">
-                  <div class="row">
-                        <div class="col-sm-6">
-                            <ol class="breadcrumb mt8">
-                                <li><a href="/my/home"><i class="fa fa-home"/></a></li>
-                                <li>Payment Methods</li>
-                            </ol>
-                        </div>
+    <!-- Delete confirm dialog -->
+    <template id="delete_payment_dialog" name="Delete Payment Method Dialog">
+        <div class="modal fade" role="dialog" aria-hidden="true" id="delete_payment_method">
+            <div class="modal-dialog">
+                <div class="modal-content">
+                    <div class="modal-header">
+                        <button type="button" class="close" data-dismiss="modal" aria-hidden="true">x</button>
+                        <h4 class="modal-titled">Delete Payment Method</h4>
                     </div>
-                    <h1>Payment Methods</h1>
-                    <h4 t-if="acquirers">Add a new payment method:</h4>
-                    <div t-if="acquirers" class="row">
-                        <div class="col-md-6">
-                            <div t-if="error_message" class="alert alert-danger">
-                              <t t-foreach="error_message" t-as="err"><t t-esc="err"/><br /></t>
-                            </div>
-                            <div t-if='len(acquirers)>1'>
-                                <label class="control-label" for="pm_acquirer_id">Acquirer</label>
-                                <select name="pm_acquirer_id" class="form-control">
-                                    <t t-foreach="acquirers" t-as="acquirer">
-                                        <option t-att-value="acquirer.id" t-esc="acquirer.name"/>
-                                    </t>
-                                </select>
+                    <div class="modal-body">
+                        <div class="row">
+                            <div class="col-md-12">
+                                <p>
+                                    Warning, Be aware that if this credit card is linked to an ongoing contract, it will prevent the payment of it.
```

Comment:
Please do the following change : https://drive.google.com/a/odoo.com/file/d/0BxqHe5rtIxY0Sk15ZGZCU2s1b3M/view?usp=drivesdk